lib/travis/api/v3/services/log/delete.rb

```diff
@@ -1,5 +1,9 @@
 module Travis::API::V3
   class Services::Log::Delete < Service
+    def self.result_type
+      :remote_log
```

Comment:
this is what I meant when I said "an odd type" above - the expected type is Log.